HW1/css/style.css

```diff
@@ -0,0 +1,66 @@
+.largetitle {
+	font-family: Verdana;
+	color: #666666;
+	font-size: 40px;
+	text-align: center;
+	margin-top: 20px;
+}
+
+
+
+.img-responsive {
+	width: 300px;
+	height: 300px;
+}
+
+.leftCol h2 {
+	font-family: Verdana;
+	color: #f07c17;
+}
+
+.leftCol ul {
+	font-family: Verdana;
+	font-size: 20px;
+	color: #666666;
+	line-height: 150%;
+	padding-top: 20px;
+}
+
+.leftCol section {
+	border-style: solid;
+	border-width: thin;
+	border-color: #666666;
+	margin-top: 1cm;
```

Comment:
You can actually write *{font-family: Verdana} to set all font inside the page to Verdana inside of  write it multiple times in every unit.